apps/core/package.json

```diff
@@ -30,6 +32,7 @@
         "react-dom": "^18.2.0"
     },
     "devDependencies": {
+        "@amplitude/analytics-types": "^0.20.0",
```

Comment:
Why only a dev dep?

Reply:
hmm, I guess perhaps this should be a `dep` instead of a `devDep` according to https://stackoverflow.com/questions/45176661/how-do-i-decide-whether-types-goes-into-dependencies-or-devdependencies, but since we don't externally publish this library + we're not exporting any of these types it doesn't really matter that much?